### PR TITLE
feat(bot-api): Add get conversation endpoints

### PR DESCRIPTION
**Pull request by lipis**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/bot-api/src/MessageHandler.ts [hunk 1]
```diff
@@ -51,6 +52,14 @@ export abstract class MessageHandler {
     }
   }
 
+  public async getConversation(conversationId: string): Promise<Conversation> {
+    return (await this.account!.service!.conversation.getConversations([conversationId]))[0];
```
Comment (ffflorian) [inline packages/bot-api/src/MessageHandler.ts]:
```suggestion
    return this.account!.service!.conversation.getConversation(conversationId);
```

File: packages/bot-api/src/MessageHandler.ts [hunk 2]
```diff
@@ -51,6 +52,14 @@ export abstract class MessageHandler {
...
+  }
+
+  public async getConversations(conversationIds?: string[]): Promise<Conversation[]> {
```
Comment (ffflorian) [inline packages/bot-api/src/MessageHandler.ts]:
```suggestion
  public getConversations(conversationIds?: string[]): Promise<Conversation[]> {
```

File: packages/bot-api/src/MessageHandler.ts [hunk 3]
```diff
@@ -51,6 +52,14 @@ export abstract class MessageHandler {
     }
   }
 
+  public async getConversation(conversationId: string): Promise<Conversation> {
```
Comment (ffflorian) [inline packages/bot-api/src/MessageHandler.ts]:
```suggestion
  public getConversation(conversationId: string): Promise<Conversation> {
```

File: packages/bot-api/src/MessageHandler.ts (hunk 1 above)
Comment (bennyn) [reply packages/bot-api/src/MessageHandler.ts]: Property 'getConversation' does not exist on type 'ConversationService'. 😈 @lipis please stick to `await` code from before.

File: packages/bot-api/src/MessageHandler.ts [hunk 4]
```diff
@@ -51,6 +52,14 @@ export abstract class MessageHandler {
     }
   }
 
+  public getConversation(conversationId: string): Promise<Conversation> {
+    return this.account!.service!.conversation.getConversation(conversationId);
```
Comment (ffflorian) [inline packages/bot-api/src/MessageHandler.ts]: This one works :)

```suggestion
    return this.account!.service!.conversation.getConversations(conversationId);
```

**Pull request merged**